tests/test_configs.py

```diff
@@ -0,0 +1,32 @@
+import os
+import re
+from glob import glob
+
+import pytest
+
+import eogrow
+from eogrow.core.config import collect_configs_from_path, interpret_config_from_dict
+from eogrow.utils.meta import load_pipeline_class
+
+CONFIG_REGEX = re.compile(r"^(?!global_).*\.json$")
+IGNORED_FOLDERS = ["other"]
+IGNORED_PIPELINES = ["eogrow.pipelines.zipmap.ZipMapPipeline"]
```

Comment:
why is this one ignored?

Reply:
No idea, I copied it from surs-geos, where it was also ignored, but I'll take another look